itou/invitations/migrations/0003_extend_all_ddets_log_invitations.py

```diff
@@ -0,0 +1,22 @@
+# Generated by Django 4.2.3 on 2023-07-24 12:14
+
+from django.db import migrations
+
+from itou.institutions.enums import InstitutionKind
+
+
+def extend_all_ddets_log_invitations(apps, _schema_editor):
+    LaborInspectorInvitation = apps.get_model("invitations", "LaborInspectorInvitation")
+    for invitation in LaborInspectorInvitation.objects.filter(institution__kind=InstitutionKind.DDETS_LOG):
+        invitation.validity_days = 90
+        invitation.save()
```

Comment:
Ne fait qu’un seul `UPDATE ... SET validity_days=90 WHERE ...` au lieu de charger individuellement chaque invitation en Python et faire un `UPDATE` par invitation.
```suggestion
    LaborInspectorInvitation.objects.filter(institution__kind=InstitutionKind.DDETS_LOG).update(invitation.validaty_days=90):
```